### ISSUE TITLE
[BUG] Predict a single sentence like "¿Recomendable? Si" returns NEG

### ISSUE THREAD

**Issue by colutti** [pysentimiento/pysentimiento#56]
**Describe the bug**

Predict a single sentence like "¿Recomendable? Si" returns 

`AnalyzerOutput(output=NEU, probas={NEU: 0.813, POS: 0.105, NEG: 0.082})`

**To Reproduce**
```python
from pysentimiento import create_analyzer
analyzer = create_analyzer(task="sentiment", lang="es")
print(analyzer.predict("¿Recomendable? Si"))
```
**Expected behavior**
It should return a higher value of POS

**Environment**

Python 3.10.9

accelerate==0.20.3
aiohttp==3.8.4
aiosignal==1.3.1
async-timeout==4.0.2
attrs==23.1.0
bandit==1.7.5
beautifulsoup4==4.12.2
blinker==1.6.2
blis==0.7.9
catalogue==2.0.8
certifi==2023.5.7
charset-normalizer==3.1.0
click==8.1.3
cmake==3.26.4
confection==0.1.0
cymem==2.0.7
datasets==2.13.1
dill==0.3.6
dnspython==2.3.0
emoji==1.7.0
es-core-news-sm @ https://github.com/explosion/spacy-models/releases/download/es_core_news_sm-3.5.0/es_core_news_sm-3.5.0-py3-none-any.whl#sha256=13b909bac0de5f5338f4327b14a9cc9736cf51f07131a3cfcb688dbeb8608bdd
filelock==3.12.2
Flask==2.3.2
frozenlist==1.3.3
fsspec==2023.6.0
gitdb==4.0.10
GitPython==3.1.31
huggingface-hub==0.15.1
idna==3.4
itsdangerous==2.1.2
Jinja2==3.1.2
joblib==1.2.0
langcodes==3.3.0
lit==16.0.6
markdown-it-py==3.0.0
MarkupSafe==2.1.3
mdurl==0.1.2
mpmath==1.3.0
multidict==6.0.4
multiprocess==0.70.14
murmurhash==1.0.9
networkx==3.1
nltk==3.8.1
numpy==1.25.0
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-cupti-cu11==11.7.101
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
nvidia-cufft-cu11==10.9.0.58
nvidia-curand-cu11==10.2.10.91
nvidia-cusolver-cu11==11.4.0.1
nvidia-cusparse-cu11==11.7.4.91
nvidia-nccl-cu11==2.14.3
nvidia-nvtx-cu11==11.7.91
packaging==23.1
pandas==2.0.3
pathy==0.10.2
pbr==5.11.1
Pillow==9.5.0
preshed==3.0.8
psutil==5.9.5
pyarrow==12.0.1
pydantic==1.10.9
Pygments==2.15.1
pymongo==4.4.0
pysentimiento==0.7.1
python-dateutil==2.8.2
pytz==2023.3
PyYAML==6.0
regex==2023.6.3
requests==2.31.0
rich==13.4.2
safetensors==0.3.1
scikit-learn==1.2.2
scipy==1.10.1
sentence-transformers==2.2.2
sentencepiece==0.1.99
six==1.16.0
smart-open==6.3.0
smmap==5.0.0
soupsieve==2.4.1
spacy==3.5.4
spacy-legacy==3.0.12
spacy-loggers==1.0.4
srsly==2.4.6
stevedore==5.1.0
sympy==1.12
thinc==8.1.10
threadpoolctl==3.1.0
tokenizers==0.13.3
torch==2.0.1
torchvision==0.15.2
tqdm==4.65.0
transformers==4.30.2
triton==2.0.0
typer==0.9.0
typing_extensions==4.6.3
tzdata==2023.3
urllib3==2.0.3
wasabi==1.1.2
Werkzeug==2.3.6
xxhash==3.2.0
yarl==1.9.2

**Comment by finiteautomata** [comment]
The sentiment analysis model was trained on Twitter data, so it is possible that some kind of these errors happen. If your data is more related to reviews, I suggest you try some Spanish model for that domain. For instance, this one?

https://huggingface.co/mrm8488/electricidad-small-finetuned-restaurant-sentiment-analysis
